### PR TITLE
Remove roles from view, fix discriminator not showing up.

### PR DESCRIPTION
**Pull request by jchristgit**
The discriminator and roles on a user's profile would not show up properly. This PR fixes the Discriminator not showing up properly and removes the role list and its accompanying template tags for now, as they didn't work properly on mobile and weren't the greatest eye catcher either.

### CODE REVIEW

File: profiles/views.py [hunk 1]
```diff
@@ -50,13 +49,10 @@ def test_func(self):
...
-            guild_id=settings.DISCORD_GUILD_ID, user_id=self.get_object().id
-        ).order_by(
-            "-role__position"
-        ).exclude(
-            role__name="@everyone"
-        )
+        try:
+            context['socialaccount'] = SocialAccount.objects.get(user=self.get_object())
+        except SocialAccount.DoesNotExist:
+            context['socialaccount'] = None
```
Comment (emmiegit) [inline profiles/views.py:55]: Don't `get()` methods usually have a way to specify a default? Then you can just have a single line.

File: profiles/templates/auth/user_detail.html [hunk 2]
```diff
@@ -18,24 +17,11 @@
...
-                   background-color:{{ role.color|int_to_css_rgba }}">
-        {{ role.name }}
-      </span>
-      {% if forloop.last %}
-        </div>
-      {% endif %}
-    {% endwith %}
-  {% endfor %}
-  <br>
+  {% if socialaccount is not None %}
```
Comment (emmiegit) [inline profiles/templates/auth/user_detail.html:20]: What is relevant about `socialaccount` here?

File: profiles/views.py (hunk 1 above)
Comment (jchristgit) [reply profiles/views.py:55]: `.get` is a method on the ORM's manager, not a dictionary, and is used to fetch a single object. There is no simpler way than this, sadly.

Comment (emmiegit) [reply profiles/views.py:55]: If this is done a lot, consider a utility function or decorator.

File: profiles/templates/auth/user_detail.html (hunk 2 above)
Comment (jchristgit) [reply profiles/templates/auth/user_detail.html:20]: The discriminator. If the user is logged in from a local account (e.g. superuser created through `manage.py`), the user wouldn't have any social account connected.

**Review by emmiegit — APPROVED** [review]

**Pull request merged**